lib/beaker-pe/install/pe_utils.rb

```diff
@@ -448,6 +455,29 @@ def deploy_frictionless_to_master(host)
           end
         end
 
+        # Check for availability of required network resources
+        # @param [Array<Host>] hosts
+        # @param [Array<String>] network_resources, defaults to NETWORK_RESOURCES
+        #
+        # @example
+        #   verify_network_resources(hosts, network_resources)
+        #
+        # @return nil
+        #
+        # @api private
+        #
+        def verify_network_resources(hosts, network_resources = NETWORK_RESOURCES)
+          logger.notify("Checking the availability of network resources.")
+          hosts.each do |host|
+            network_resources.each do |resource|
+              on host, "curl -s #{resource} > /dev/null", :acceptable_exit_codes => [0,6]
```

Comment:
Should we add 35 to the exit codes?

Reply:
yeah, in fact we should probably add a wildcard since this should be non-blocking.